### ISSUE TITLE
Crew monitoring console is blank

### ISSUE THREAD

**Issue by Rad-Paperstock** [NovaSector/NovaSector#370]
Reporting client version: 514.1589

## Issue Summary

there is no crew vitals on the crew monitoring console and handheld console! It was working not too long ago in the shift, but now it's all gone. I am not certain this will happen again next round, but for now it's blank. 

## Round ID:
[10484](https://scrubby.melonmesa.com/round/10484)

## Testmerges:
- [Antag optin ](https://github.com/NovaSector/NovaSector/pull/121)
- [[TM ONLY] [EARLY MIRROR] Revert "Makes immerse use weakrefs", prioritizing some ref removal lines instead](https://github.com/NovaSector/NovaSector/pull/238)
- [[MODULAR] Removes Arbitrary Loadout Restrictions](https://github.com/NovaSector/NovaSector/pull/58)
- [[MODULAR] Chat color tweaks!](https://github.com/NovaSector/NovaSector/pull/323)
- [Kahraman, after collecting dust for 100 years, finally has its own content, hear ye guncargo naysayers and weep](https://github.com/NovaSector/NovaSector/pull/98)
- [[TM-ONLY] Testing chatmessage debug](https://github.com/NovaSector/NovaSector/pull/275)
- [[Testing] Fixing a potential issue with liquid turfs](https://github.com/NovaSector/NovaSector/pull/356)

## Reproduction:

Simply put, I just opened up the console UI for crew monitoring, and it's not working. I am uncertain if there are other consoles with the same error.
